src/app/views/marketplace/marketplace-app.controller.coffee

```diff
@@ -169,6 +169,7 @@ angular.module 'mnoEnterpriseAngular'
           (response) ->
             vm.reviews.list.splice(key, 1)
             updateAverageRating(response.average_rating)
+            vm.reviews.pageChangedCb(vm.app.id, vm.reviews.nbItems, vm.reviews.page = 1)
```

Comment:
This will refresh the list calling the backend. To have a faster UI, we could use used `splice` or `unshift` methods.